scss/bootstrap/_modal.scss

```diff
@@ -16,7 +16,7 @@
   right: 0;
   bottom: 0;
   left: 0;
-  z-index: $zindex-modal;
+  z-index: 1050;
```

Comment:
Where does 1050 come from?

Reply:
The value of `$zindex-modal` has been replaced from core Moodle's `_variables.scss`. I have only replaced values directly of the variables we use for File-picker rather than importing entire SASS file.